pages/common/ruff.md

```diff
@@ -0,0 +1,12 @@
+# ruff
+
+> An extremely fast Python linter and code formatter, written in Rust.
```

Comment:
```suggestion
> An extremely fast Python linter and code formatter written in Rust.
```

Reply:
I copied this from [the official GitHub repo](https://github.com/astral-sh/ruff).